lib/src/rules/cascade_invocations.dart

```diff
@@ -51,6 +52,14 @@ someReference
 
 ''';
 
+Element _findElement(SimpleIdentifier identifier) {
+  if (identifier.bestElement is PropertyAccessorElement) {
```

Comment:
style: If you assign "identifier.bestElement" to a local variable, then (a) you won't have to repeat the expression so often and (b) you won't need the cast on the next line.

This probably needs to only return the variable when the accessor is synthetic, otherwise code like the following might start generating lints:
```
Foo get createFoo => new Foo();

void m() {
  createFoo.a();
  createFoo.b();
}
```
Which, as written would create two separate instances of Foo, but if converted to a cascade would create a single instance and invoke both method on it.